Common subdirectories: openssl-0.9.8zc/apps and openssl-0.9.8zc-patched/apps
Common subdirectories: openssl-0.9.8zc/bugs and openssl-0.9.8zc-patched/bugs
Common subdirectories: openssl-0.9.8zc/certs and openssl-0.9.8zc-patched/certs
diff -au openssl-0.9.8zc/Configure openssl-0.9.8zc-patched/Configure
--- openssl-0.9.8zc/Configure	2014-10-15 16:45:00.000000000 +0400
+++ openssl-0.9.8zc-patched/Configure	2014-10-22 13:53:05.965106566 +0400
@@ -1,4 +1,4 @@
-:
+#!/usr/bin/perl
 eval 'exec perl -S $0 ${1+"$@"}'
     if $running_under_some_shell;
 ##
@@ -556,6 +556,8 @@
 ##### Compaq Non-Stop Kernel (Tandem)
 "tandem-c89","c89:-Ww -D__TANDEM -D_XOPEN_SOURCE -D_XOPEN_SOURCE_EXTENDED=1 -D_TANDEM_SOURCE -DB_ENDIAN::(unknown):::THIRTY_TWO_BIT:::",
 
+########## Linux-arc ###########
+"linux-arc","gcc:-DL_ENDIAN -DTERMIO -O3 -fomit-frame-pointer -Wall::-D_REENTRANT::-ldl:BN_LLONG::::::::::::dlfcn:linux-shared:-fPIC::.so.\$(SHLIB_MAJOR).\$(SHLIB_MINOR)",
 );
 
 my @MK1MF_Builds=qw(VC-WIN64I VC-WIN64A
Common subdirectories: openssl-0.9.8zc/crypto and openssl-0.9.8zc-patched/crypto
Common subdirectories: openssl-0.9.8zc/demos and openssl-0.9.8zc-patched/demos
Common subdirectories: openssl-0.9.8zc/doc and openssl-0.9.8zc-patched/doc
Common subdirectories: openssl-0.9.8zc/engines and openssl-0.9.8zc-patched/engines
Common subdirectories: openssl-0.9.8zc/fips and openssl-0.9.8zc-patched/fips
Common subdirectories: openssl-0.9.8zc/include and openssl-0.9.8zc-patched/include
Common subdirectories: openssl-0.9.8zc/MacOS and openssl-0.9.8zc-patched/MacOS
diff -au openssl-0.9.8zc/Makefile.org openssl-0.9.8zc-patched/Makefile.org
--- openssl-0.9.8zc/Makefile.org	2014-10-15 16:45:00.000000000 +0400
+++ openssl-0.9.8zc-patched/Makefile.org	2014-10-22 13:54:14.249104570 +0400
@@ -615,7 +615,7 @@
 dist_pem_h:
 	(cd crypto/pem; $(MAKE) -e $(BUILDENV) pem.h; $(MAKE) clean)
 
-install: all install_docs install_sw
+install: all install_sw
 
 install_sw:
 	@$(PERL) $(TOP)/util/mkdir-p.pl $(INSTALL_PREFIX)$(INSTALLTOP)/bin \
Common subdirectories: openssl-0.9.8zc/ms and openssl-0.9.8zc-patched/ms
Common subdirectories: openssl-0.9.8zc/Netware and openssl-0.9.8zc-patched/Netware
Common subdirectories: openssl-0.9.8zc/os2 and openssl-0.9.8zc-patched/os2
Common subdirectories: openssl-0.9.8zc/perl and openssl-0.9.8zc-patched/perl
Common subdirectories: openssl-0.9.8zc/shlib and openssl-0.9.8zc-patched/shlib
Common subdirectories: openssl-0.9.8zc/ssl and openssl-0.9.8zc-patched/ssl
Common subdirectories: openssl-0.9.8zc/test and openssl-0.9.8zc-patched/test
Common subdirectories: openssl-0.9.8zc/times and openssl-0.9.8zc-patched/times
Common subdirectories: openssl-0.9.8zc/tools and openssl-0.9.8zc-patched/tools
Common subdirectories: openssl-0.9.8zc/util and openssl-0.9.8zc-patched/util
Common subdirectories: openssl-0.9.8zc/VMS and openssl-0.9.8zc-patched/VMS
